content/zh/about/feature-stages/index.md

```diff
@@ -44,7 +44,7 @@ Below is our list of existing features and their current phases. This informatio
 | Gateway: Ingress, Egress for all protocols | Stable
 | TLS termination and SNI Support in Gateways | Stable
 | SNI (multiple certs) at ingress | Stable
-| [Locality load balancing](/zh/docs/ops/configuration/traffic-management/locality-load-balancing/) | Beta
+| [Locality load balancing](/zh/docs/tasks/traffic-management/locality-load-balancing/) | Beta
```

Comment:
You shouldn't change the zh files in en PRs. This is now a broken link because you haven't moved the translated version yet. We don't detect broken zh links because the build is only doing lint checking on en files, because we're not supposed to touch the zh files, the translation team will fix up the zh files later.